packages/@dcl/ecs/src/systems/crdt/transports/rendererTransport.ts

```diff
@@ -7,6 +7,13 @@ const componentIds = Object.values(ECSComponentIDs)
   .map(Number)
 
 export function createRendererTransport(): Transport {
+  if (typeof globalThis.dcl === 'undefined') {
+    // TODO: replace with new rpc
+    throw new Error(
```

Comment:
![image](https://user-images.githubusercontent.com/8042536/187291127-3e974e05-ae95-4606-891e-06a01fafce68.png)

Reply:
change `globalThis.dcl` to just `dcl` as in `initialization.ts:17`

Reply:
I approved the PR, but this should be changed, I test modifying the `game.js` and it works perfectly!